### DISCUSSION TITLE
Laravel Envoy - Started Compiler

### DISCUSSION THREAD

**Discussion by amandiobm** [laravel/framework#32930] [General]
Hey guys, I want to send a notification always my envoy runs. Reading the docs, it only has the `@finished` directive/compiler.

```
@finished
    @slack('webhook-url', '#bots')
@endfinished
```

I'm looking for something similar.

```
@started
    @slack('webhook-url', '#bots')
@endstarted
```

There is any thing liked?!

**Comment by royduin** [comment]
Did you read the docs? There is a `@setup` directive which can be used: https://laravel.com/docs/7.x/envoy#setup

Looking for all the available directives? Search for `finished` in the Envoy repo: https://github.com/laravel/envoy/search?q=finished&unscoped_q=finished, the second result is the compiler which has a array with all available options: https://github.com/laravel/envoy/blob/1b2d391ed0009090546f83e0e4b83590fc6ec8fe/src/Compiler.php#L16:L48
